calico/reference/public-cloud/ibm.md

```diff
@@ -4,15 +4,12 @@ description: Calico integration with IBM Cloud.
 canonical_url: '/reference/public-cloud/ibm'
 ---
 
-{{site.prodname}} is installed and configured automatically in your [IBM Cloud Kubernetes Service][IBMKUBE].  Default policies are created to protect your Kubernetes cluster, with the option to create your own policies to protect specific services.
+{{site.prodname}} is installed and configured automatically in your [IBM Cloud Kubernetes Service][https://www.ibm.com/cloud/container-service/].  Default policies are created to protect your Kubernetes cluster, with the option to create your own policies to protect specific services.
 
 ## IP-in-IP encapsulation
 
-[IP-in-IP encapsulation][IPIP] is automatically configured to only encapsulate packets traveling across subnets, and uses NAT for outgoing connections from your containers.
+[IP-in-IP encapsulation][{{ site.baseurl }}/networking/vxlan-ipip] is automatically configured to only encapsulate packets traveling across subnets, and uses NAT for outgoing connections from your containers.
```

Comment:
[IP-in-IP encapsulation][{{ site.baseurl }}/networking/vxlan-ipip] is automatically configured to encapsulate packets traveling only across subnets, and to use NAT for outgoing connections from your containers.